OpenRA.Mods.RA2/Traits/BaseSpawnerParent.cs

```diff
@@ -202,14 +202,17 @@ void INotifyActorDisposing.Disposing(Actor self)
 		public virtual void SpawnIntoWorld(Actor self, Actor child, WPos centerPosition)
 		{
 			var exit = self.RandomExitOrDefault(self.World, null);
-			SetSpawnedFacing(child, self, exit);
+			SetSpawnedFacing(child, exit);
 
 			self.World.AddFrameEndTask(w =>
 			{
 				if (self.IsDead)
 					return;
 
-				var spawnOffset = exit == null ? WVec.Zero : exit.Info.SpawnOffset;
+				var spawnOffset = WVec.Zero;
+				if (exit != null)
+					spawnOffset = exit.Info.SpawnOffset.Rotate(new WRot(WAngle.Zero, WAngle.Zero, WAngle.FromFacing(64 - facing.Facing)));
```

Comment:
I could move this to the Exit trait as that appears to maintain the facing?

Reply:
I think that would indeed be better. Where is the magic `64 -` coming from?

Reply:
I can confirm that this also works in a 2D plane with only sprites. Just had to change offset from `-250` to `250` so the minus sign may be "wrong" compared to `Armament` `LocalOffset` from a regular bullet.